### DISCUSSION TITLE
Cannot see the button for upload proof of evidence for verify github education

### DISCUSSION THREAD

**Discussion by congtran201290** [community/community#137461] [GitHub Education]
### Hi everyone,

Hello Team,
My name is Cong and I am applying to the Github copilot for the Education students.
Related to application process, in items:
- Please upload proof of your academic status.
- Upload an image supporting your reason for being far from campus.
I cannot find the button where can I upload for my pdf file to show as evidence 
Please advice me how to upload the file
Thank a lot, 
![proof of students](https://github.com/user-attachments/assets/2472c7c5-9f04-45f2-862a-b9e6f988d8f3)

**Comment by md-ashraful-islam-ullas** [comment]
The option is not available.  You can use OBS studio.  (I got rejected using this method)

Or I guess this Screenshot taken from computer. Apply using your mobile phone. You will see front camera and back camera option when applying from mobile phone. Open your PDF or document in your computer.
 Open your back Camera and take photo from computer. (I got approved in this system)

**Reply by iabdo9** [reply]
my back camera doesn't work
when I click back camera nothing happen

**Reply by congtran201290** [reply]
Hello @md-ashraful-islam-ullas ,
thank you a lot for your advice, which is helpful for me a lot.

Hello @iabdo9 ,
yes, you are correct, only front camera is available, we cannot select the back camera as well.
